SqlFundamentalsDemo/SqlDemo/EmployeeInfo.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE VIEW [dbo].[EmployeeInfo]
+	AS SELECT 
+    E.Id AS EmployeeId,
```

Comment:
It's always a good practice to make correct tabulation for your SQL script, it really simplifies readability. Let's, please, make the same indentation for `E.Id` as same as for `ISNULL`